### PR TITLE
Starting New features for Telco Core 413 RN

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1123]: Document new features

Version(s):
4.13 

Issue:
https://issues.redhat.com/browse/TELCODOCS-1123

Link to docs preview:

- https://57258--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-supporting-ocp-nodes-with-dual-port-nics
- https://57258--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-configuring-power-states-using-ztp

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc
Comment (tmalove) [inline release_notes/ocp-4-13-release-notes.adoc:278]:
```suggestion
You can now create a password for the {op-system} `core` user. If you cannot use SSH or the `oc debug node` command to access a node, this password allows you to use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC).
```
See the IBMSG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-13-release-notes.adoc
Comment (kquinn1204) [reply release_notes/ocp-4-13-release-notes.adoc:278]: I will update but to be honest this is not content I created.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (sakhoury) [inline release_notes/ocp-4-13-release-notes.adoc:387]: Please reword this to indicate that OCP 4.12 had introduced the ability to set power states for critical and non-critical workloads. This release now enables the user to configure power states using ZTP.

**Review by sakhoury — CHANGES_REQUESTED** [review]
Reviewing `Configuring power states using zero touch provisioning (ZTP)` only.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (kquinn1204) [reply release_notes/ocp-4-13-release-notes.adoc:387]: Resolved by comment below

**Comment by kquinn1204** [comment]
@mjpytlak This can hopefully be merged now

**Comment by mjpytlak** [comment]
@kquinn1204 I believe we are still waiting on @sakhoury approval.

File: release_notes/ocp-4-13-release-notes.adoc
Comment (mjpytlak) [inline release_notes/ocp-4-13-release-notes.adoc:389]:
```suggestion
For more information about the feature, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-using-pgt-to-configure-power-saving-states_ztp-advanced-policy-config[Configuring power states using PolicyGenTemplates CRs].
```

**Comment by kquinn1204** [comment]
@mjpytlak Thank you got LGTM and fixed that mistake so should be OK fingers crossed

File: release_notes/ocp-4-13-release-notes.adoc
Comment (mjpytlak) [inline release_notes/ocp-4-13-release-notes.adoc:256]:
```suggestion
For more information about installing {product-title} on nodes with dual-port NICs, see xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#nw-sriov-dual-nic-con_preparing-to-install-on-bare-metal[NIC partitioning for SR-IOV devices].
```
@kquinn1204 Sorry man. This one got by me the last time.

Comment (kquinn1204) [reply release_notes/ocp-4-13-release-notes.adoc]: Sorry my bad I should have noticed that @mjpytlak hopefully ok now

**Pull request merged**
